relay-server/src/services/processor/replay.rs

```diff
@@ -88,6 +89,13 @@ pub fn process(
                 item.set_payload(ContentType::OctetStream, replay_recording);
             }
             ItemType::ReplayVideo => {
+                // If video replays have not been enabled and a video replay was received
+                // drop the envelope.
+                if !video_replay_enabled {
+                    state.managed_envelope.drop_items_silently();
+                    return Ok(());
+                }
```

Comment:
nit: the comment is not helpful as it describes what the `match` variant and `if` condition match on, so I'd suggest deleting it.